java/src/jmri/jmrix/dccpp/DCCppPacketizer.java

```diff
@@ -75,6 +76,11 @@ protected int addHeaderToOutput(byte[] msg, jmri.jmrix.AbstractMRMessage m) {
         return 1;
     }
 
+    public void startThreads() {
+	// Doesn't do anything generically.  Here for TCP version.
+	// Not abstract because I don't want to have to deal with that.
```

Comment:
If this method *should* be abstract (as your comments indicate), make it abstract.